vuestorefront/magento2
# Develop

## Description
Fix missing search results localisation

## Related Issue
#405

## Motivation and Context
Bugfix

## How Has This Been Tested?

- Check search results linking
- Search in code for all non-localized:link bindings

## Screenshots (if appropriate):
-

## Types of changes


## Checklist:
